components/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { ReactComponent as Checked } from '../assets/images/checked.svg'
+
+type Props = {
+  children?: React.ReactNode
+  success?: boolean
+  text?: string
+  title: string
```

Comment:
I would also accept variable columns in case we want to customize it in the future:

```
col: number
offset: number
```